content/swift/concepts/protocols/terms/encodable/encodable.md

```diff
@@ -1,20 +1,28 @@
 ---
-Title: 'Encodable'
```

Comment:
The changes to the metadata break the entry and should be removed. You should review the rules on metadata [here](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md#metadata) before attempting to edit another entry's metadata. Please revert these changes to the metadata.

Reply:
I restored the whole section. Sorry for distracting the intended structure. I was focused on the layout and unfortunately did not realize breaking the entry's metadata.